### PR TITLE
Update sitemap.xml.php

### PR DESCRIPTION
**Pull request by gejobj**
Fixed Google Search Console date format error. lastmod date changed from +0100 to +01:00

### CODE REVIEW

File: phpmyfaq/sitemap.xml.php [hunk 1]
```diff
@@ -65,7 +65,11 @@ function buildSiteMapNode($location, $lastModified = null)
         $lastModified = Date::createIsoDate($_SERVER['REQUEST_TIME'], DATE_W3C, false);
     }
     if (preg_match('/^[1|2][0-9]{3}-[0|1][0-9]-[0|1|2|3][0-9]$/', $lastModified)) {
-        $lastModified .= 'T'.date('H:i:sO');
+        $lastModified .= 'T'.date('H:i:sP');
+    } else if ( preg_match('/([\+|\-][0-9]{2})([0-9]{2})$/', $lastModified, $arr_find) ) {
```
Comment (thorsten) [inline phpmyfaq/sitemap.xml.php:69]: Could you please rename `$arr_find` to `$arrayFind`? Thanks!

**Review by thorsten — CHANGES_REQUESTED** [review]
Could you please also add this for the 3.0 branch as well? Thank you?

File: phpmyfaq/sitemap.xml.php (hunk 1 above)
Comment (gejobj) [reply phpmyfaq/sitemap.xml.php:69]: Yes, of course :-)

**Comment by gejobj** [comment]
Here you have: https://github.com/thorsten/phpMyFAQ/pull/1844
:-)

**Pull request closed without merging**